src/core/variables.tf

```diff
@@ -628,7 +628,7 @@ variable "cosmosdb_mongodb_enable_autoscaling" {
 variable "cosmosdb_mongodb_max_throughput" {
   type        = number
   description = "The maximum throughput of the MongoDB database (RU/s). Must be between 4,000 and 1,000,000. Must be set in increments of 1,000. Conflicts with throughput"
-  default     = 4000
+  default     = 1000
```

Comment:
As stated by the description, and suggestede from the failing code review, the minimun value is 4000, as already set

Reply:
ok we need also to upgrade terraform provider 3.2.0...
for the moment we'll make this pr as draft

https://registry.terraform.io/providers/hashicorp/azurerm/3.2.0/docs/resources/cosmosdb_sql_container#max_throughput